### PR TITLE
[i18n-DE] Complete first toc chapter

### PR DESCRIPTION
**Pull request by flozi00**
# What does this PR do ?
This PR translates all the file of the first toc chapter to german
It continues https://github.com/huggingface/transformers/issues/18564

- [x] add new model
- [x] add new pipeline
- [x] add tensorflow model
- [x] llm tutorial
- [x] peft
- [x] run scripts
- [x] transformers agents
- [x] re-read at the live documentation preview from PR

### CODE REVIEW

**Comment by flozi00** [comment]
@ArthurZucker or who is the correct one to ping ?

**Review by ArthurZucker — COMMENTED** [review]
Hey! Thanks a lot for these translations 😉 could you make sure the PR descriptions looks like: #25877? 
Pinging @stevhliu for doc review!

File: docs/source/de/peft.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+```py
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+peft_model_id = "ybelkada/opt-350m-lora"
+model = AutoModelForCausalLM.from_pretrained(peft_model_id, device_map="auto", load_in_8bit=True)
+```
+
+## Einen neuen Adapter hinzufügen
+
+Sie können [~peft.PeftModel.add_adapter`] verwenden, um einen neuen Adapter zu einem Modell mit einem bestehenden Adapter hinzuzufügen, solange der neue Adapter vom gleichen Typ ist wie der aktuelle Adapter. Wenn Sie zum Beispiel einen bestehenden LoRA-Adapter an ein Modell angehängt haben:
```
Comment (stevhliu) [inline docs/source/de/peft.md:97]:
```suggestion
Sie können [`~peft.PeftModel.add_adapter`] verwenden, um einen neuen Adapter zu einem Modell mit einem bestehenden Adapter hinzuzufügen, solange der neue Adapter vom gleichen Typ ist wie der aktuelle Adapter. Wenn Sie zum Beispiel einen bestehenden LoRA-Adapter an ein Modell angehängt haben:
```

File: docs/source/de/peft.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+```
+
+Um einen neuen Adapter hinzuzufügen:
+
+```py
+# attach new adapter with same config
+model.add_adapter(lora_config, adapter_name="adapter_2")
+```
+
+Jetzt können Sie mit [~peft.PeftModel.set_adapter`] festlegen, welcher Adapter verwendet werden soll:
```
Comment (stevhliu) [inline docs/source/de/peft.md:121]:
```suggestion
Jetzt können Sie mit [`~peft.PeftModel.set_adapter`] festlegen, welcher Adapter verwendet werden soll:
```

File: docs/source/de/peft.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+PEFT-Adapter werden von der Klasse [`Trainer`] unterstützt, so dass Sie einen Adapter für Ihren speziellen Anwendungsfall trainieren können. Dazu müssen Sie nur ein paar weitere Codezeilen hinzufügen. Zum Beispiel, um einen LoRA-Adapter zu trainieren:
+
+<Tip>
+
+Wenn Sie mit der Feinabstimmung eines Modells mit [`Trainer`] noch nicht vertraut sind, werfen Sie einen Blick auf das Tutorial [Feinabstimmung eines vortrainierten Modells](Training).
+
+</Tip>
+
+1. Definieren Sie Ihre Adapterkonfiguration mit dem Aufgabentyp und den Hyperparametern (siehe [~peft.LoraConfig`] für weitere Details darüber, was die Hyperparameter tun).
```
Comment (stevhliu) [inline docs/source/de/peft.md:177]:
```suggestion
1. Definieren Sie Ihre Adapterkonfiguration mit dem Aufgabentyp und den Hyperparametern (siehe [`~peft.LoraConfig`] für weitere Details darüber, was die Hyperparameter tun).
```

File: docs/source/de/run_scripts.md [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+
+Für jede Funktion, die Sie in einem Beispielskript implementieren möchten, diskutieren Sie bitte im [Forum] (https://discuss.huggingface.co/) oder in einem [issue] (https://github.com/huggingface/transformers/issues), bevor Sie einen Pull Request einreichen. Wir freuen uns zwar über Fehlerkorrekturen, aber es ist unwahrscheinlich, dass wir einen Pull Request zusammenführen, der mehr Funktionalität auf Kosten der Lesbarkeit hinzufügt.
+
+Diese Anleitung zeigt Ihnen, wie Sie ein Beispiel für ein Trainingsskript zur Zusammenfassung in [PyTorch](https://github.com/huggingface/transformers/tree/main/examples/pytorch/summarization) und [TensorFlow](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/summarization) ausführen können. Sofern nicht anders angegeben, sollten alle Beispiele mit beiden Frameworks funktionieren.
+
+## Einrichtung
+
+Um die neueste Version der Beispielskripte erfolgreich auszuführen, müssen Sie 🤗 Transformers aus dem Quellcode** in einer neuen virtuellen Umgebung installieren:
```
Comment (stevhliu) [inline docs/source/de/run_scripts.md:31]:
```suggestion
Um die neueste Version der Beispielskripte erfolgreich auszuführen, **müssen Sie 🤗 Transformers aus dem Quellcode** in einer neuen virtuellen Umgebung installieren:
```

File: docs/source/de/run_scripts.md [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+    --dataset_config "3.0.0" \
+    --source_prefix "summarize: " \
+    --output_dir /tmp/tst-summarization \
+    --per_device_train_batch_size=4 \
+    --per_device_eval_batch_size=4 \
+    --overwrite_output_dir \
+    --predict_with_generate
+```
+
+TensorFlow-Skripte verwenden eine [`MirroredStrategy`] (https://www.tensorflow.org/guide/distributed_training#mirroredstrategy) für verteiltes Training, und Sie müssen dem Trainingsskript keine zusätzlichen Argumente hinzufügen. Das TensorFlow-Skript verwendet standardmäßig mehrere GPUs, wenn diese verfügbar sind.
```
Comment (stevhliu) [inline docs/source/de/run_scripts.md:149]:
```suggestion
TensorFlow-Skripte verwenden eine [`MirroredStrategy`](https://www.tensorflow.org/guide/distributed_training#mirroredstrategy) für verteiltes Training, und Sie müssen dem Trainingsskript keine zusätzlichen Argumente hinzufügen. Das TensorFlow-Skript verwendet standardmäßig mehrere GPUs, wenn diese verfügbar sind.
```

File: docs/source/de/transformers_agents.md [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+```py
+agent.run("Draw me a picture of the sea then transform the picture to add an island")
+```
+
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/sea_and_island.png" width=200>
+
+<br/>
+
+
+Jede [~Agent.run] Operation ist unabhängig, so dass Sie sie mehrmals hintereinander mit unterschiedlichen Aufgaben ausführen können.
```
Comment (stevhliu) [inline docs/source/de/transformers_agents.md:147]:
```suggestion
Jede [`~Agent.run`] Operation ist unabhängig, so dass Sie sie mehrmals hintereinander mit unterschiedlichen Aufgaben ausführen können.
```

File: docs/source/de/transformers_agents.md [hunk 7]
```diff
@@ -0,0 +1,323 @@
...
+```py
+agent.run("Draw me a picture of the `prompt`", prompt="a capybara swimming in the sea")
+```
+
+</Tip>
+
+
+### Chat-basierte Ausführung (Chat)
+
+Der Agent verfügt auch über einen Chat-basierten Ansatz, der die Methode [~Agent.chat] verwendet:
```
Comment (stevhliu) [inline docs/source/de/transformers_agents.md:185]:
```suggestion
Der Agent verfügt auch über einen Chat-basierten Ansatz, der die Methode [`~Agent.chat`] verwendet:
```

**Review by stevhliu — APPROVED** [review]
Great job translating these! Once we fix the formatting we can merge :)

**Pull request merged**